code/game/machinery/PDApainter.dm

```diff
@@ -10,71 +10,71 @@
 	var/obj/item/card/id/storedid = null
 	var/pda_icons = list(
 		"neutral" = "pda",
-		"Assistant" = "pda-assistant",
-		"Atmospheric Technician" = "pda-atmos",
-		"Bartender" = "pda-bartender",
+		JOB_ASSISTANT = "pda-assistant",
+		JOB_ATMOSPHERIC_TECHNICIAN = "pda-atmos",
+		JOB_BARTENDER = "pda-bartender",
 		"Service" = "pda-service",
-		"Captain" = "pda-captain",
-		"Cargo Technician" = "pda-cargo",
-		"Chaplain" = "pda-chaplain",
-		"Chemist" = "pda-chemistry",
-		"Chief Medical Officer" = "pda-cmo",
-		"Chief Engineer" = "pda-ce",
-		"Clown" = "pda-clown",
-		"Cook" = "pda-cook",
-		"Curator" = "pda-library",
-		"Detective" = "pda-detective",
+		JOB_CAPTAIN = "pda-captain",
+		JOB_CARGO_TECHNICIAN = "pda-cargo",
+		JOB_CHAPLAIN = "pda-chaplain",
+		JOB_CHEMIST = "pda-chemistry",
+		JOB_CHIEF_MEDICAL_OFFICER = "pda-cmo",
+		JOB_CHIEF_ENGINEER = "pda-ce",
+		JOB_CLOWN = "pda-clown",
+		JOB_COOK = "pda-cook",
+		JOB_CURATOR = "pda-library",
+		JOB_DETECTIVE = "pda-detective",
 		"Engineer" = "pda-engineer",
```

Comment:
Missed one? if not on purpose you also missed Scientist here

Reply:
This `Engineer` means both of Station engi and atmos tech (like the Service one). but scientist is a good catch. thanks.